src/components/Translate.js

```diff
@@ -15,26 +15,63 @@ const Translate = ({ keyName, data, ...props }) => (
   <Text {...props}>{suppplant(strings[keyName], data)}</Text>
 )
 
-export const Title = ({ style, ...props }) => (
-  <Translate {...props} style={[styles.title].concat(style)} />
+const createStyledTranslation = (styled) => ({ keyName, data, text, style, ...props }) => (
+  text ? (
+    <Text {...props}
+      style={[styles.text].concat(styled).concat(style)} >
```

Comment:
you can do this with only one concat
```suggestion
      style={[styles.text].concat(styled, style)} >
```

Reply:
actually, I think is better if we make one of this concat when the component is created, like this:
```js
export const Heading1 = createStyledTranslation([styles.text, styles.heading1])
```
and here you only concat the custom styles
```suggestion
      style={defaultStyles).concat(style)} >
```

Reply:
Done!